gst/nnstreamer/tensor_common.c

```diff
@@ -156,6 +156,31 @@ gst_tensor_info_is_equal (const GstTensorInfo * i1, const GstTensorInfo * i2)
   return TRUE;
 }
 
+/**
+ * @brief Copy tensor info
+ * @note GstTensorInfo::name should be freed with g_free()
+ */
+void
+gst_tensor_info_copy (GstTensorInfo * dest, const GstTensorInfo * src)
+{
+  guint i;
+
+  g_return_if_fail (dest != NULL);
+  g_return_if_fail (src != NULL);
+
+  gst_tensor_info_init (dest);
```

Comment:
You do not need init. Line 173:181 overwrites ALL properties.

Reply:
modified. if src-name is not null, copy string.